docs/topics/braintree.rst

```diff
@@ -102,6 +97,8 @@ object and Braintree customer.
     :query buyer: the primary key of the buyer.
     :query active: the active status.
 
+.. _payment-methods-label:
```

Comment:
it is, but i've found being able to grep label and find all the labels for cross referencing really useful